scripts/microshift-disable.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
The name of the script may be a bit misleading as it does not disable microshift, only stops services/pods and reverts OVN configuration. MicroShift will be started after boot.